scripts/qesap/pylint.rc

```diff
@@ -161,7 +161,8 @@ disable=line-too-long,
         unused-argument,
         too-many-locals,
         too-many-arguments,
-        duplicate-code
+        duplicate-code,
+	invalid-name
```

Comment:
What was the warning to suppress?

Reply:
> What was the warning to suppress?

The scripts in `ansible` subdirectory.  But I would fix this only if there's the will to fix those scripts.

Reply:
Isn't it better than to ignore all `invalid-name` errors to simply not check the ansible sub-directory?

Reply:
> Isn't it better than to ignore all `invalid-name` errors to simply not check the ansible sub-directory?

Dropped the whole commit.